src/config.json

```diff
@@ -19,7 +19,9 @@
         "extension_registry": "https://s3.amazonaws.com/extend.brackets/registry.json",
         "extension_url": "https://s3.amazonaws.com/extend.brackets/{0}/{0}-{1}.zip",
         "linting.enabled_by_default": true,
-        "build_timestamp": ""
+        "build_timestamp": "",
+        "healthDataServerURL" : "http://10.192.16.236:8080",
+        "healthDataTestServerURL" : "http://10.192.16.236:8080?test=true"
```

Comment:
@peterflynn @abose We will not be needing testServerURL now. We have infrastructure to test it, so not required here.